Yet another Linux build failure (git head)

**SLADE Version**: (git head)
**Operating System**: (Gentoo Linux)
**Issue Details**:
[ 48%] Building CXX object src/CMakeFiles/slade.dir/Dialogs/Preferences/EditingPrefsPanel.o
In file included from /mnt/data/source/system/SLADE/src/Dialogs/Preferences/EditingPrefsPanel.h:5,
                 from /mnt/data/source/system/SLADE/src/Dialogs/Preferences/EditingPrefsPanel.cpp:33:
/mnt/data/source/system/SLADE/src/./UI/Lists/VirtualListView.h:82:18: error: 'wxListItemAttr* VirtualListView::OnGetItemColumnAttr(long int, long int) const' marked 'override', but does not override
  wxListItemAttr* OnGetItemColumnAttr(long item, long column) const override
                  ^~~~~~~~~~~~~~~~~~~
make[2]: *** [src/CMakeFiles/slade.dir/build.make:792: src/CMakeFiles/slade.dir/Dialogs/Preferences/EditingPrefsPanel.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:163: src/CMakeFiles/slade.dir/all] Error 2
make: *** [Makefile:130: all] Error 2

--- a/src/UI/Lists/VirtualListView.h
+++ b/src/UI/Lists/VirtualListView.h
@@ -79,11 +79,13 @@ protected:
 		return item_attr_.get();
 	}
 
+#ifdef __WXMSW__
 	wxListItemAttr* OnGetItemColumnAttr(long item, long column) const override
 	{
 		updateItemAttr(item, column, itemIndex(item));
 		return item_attr_.get();
 	}
+#endif // __WXMSW__
 
 	// Events
 	void onColumnResize(wxListEvent& e);

